### DISCUSSION TITLE
Aquarite SWG Temperature and Flow Feedback?

### DISCUSSION THREAD

**Discussion by 643452** [tagyoureit/nodejs-poolController#945] [Q&A]
Hello,

I'm debating on if it would be worth it to setup this control system for my Hayward Aquarite SWG. On the front panel of the Aquarite, you can see pool temperature and there is an LED/alarm for no flow. 
Do you get these two items back with this nodejs poolController? From the wiki I can see an error byte with a no flow error but I'm unsure how that behaves in its entirety (for example can I extract just flow status and ignore other errors). I don't see anything in the wiki about temperature feedback

Anyone who has set this up with an Aquarite SWG know the answer? Much appreciated!!

**Comment by tagyoureit** [comment]
> if it would be worth it to setup this control system for my Hayward Aquarite SWG

There's no real downside to setting it up.  If you currently "set and forget" the output % then adding this won't likely give you much additional value.  But, if you want to control the chlorinator and get back the status, salt readings, output % then this will add value.

You do _not_ get back the pool temperature from the chlorinator.
You do get back an error if there is no flow.

> but I'm unsure how that behaves in its entirety (for example can I extract just flow status and ignore other errors)

There isn't anything more to this than the chlorinator reporting it's status.  The chlor has a built-in water flow sensor which acts as a safe guard so it isn't making chlorine gas when there is no flow.  But we don't actually do anything with this other than display it on the dashboard.
